charts/oes/values.yaml

```diff
@@ -78,6 +78,11 @@ global:
       tls:
         secretName: gate-authtls
 
+  ## Open Ldap password required for spin-cli for installing the sample pipelines
+  openldap:
```

Comment:
We are duplicating openldap config here, we should try to configure once and use everywhere